pkgs/top-level/aliases.nix

```diff
@@ -1136,6 +1136,7 @@ mapAliases ({
   squid4 = squid;  # added 2019-08-22
   sshfsFuse = sshfs-fuse; # added 2016-09
   stanchion = throw "Stanchion was part of riak-cs which is not maintained anymore"; # added 2020-10-14
+  steam-run-native = throw "steam-run-native has been removed as it's now the same as steam-run"; # added 2022-02-21
```

Comment:
Why not just alias to `steam-run`?

Reply:
I agree with the change, on one hand it will require users to fix their scripts, but on the other hand this will prevent legacy. There is no reason to keep a working `steam-run-native` command if it's a symlink to another command.

Reply:
Yea, I guess there's not much harm of just aliasing it.